src/AppBundle/Resources/translations/report-prof-deputy-costs-estimate.en.yml

```diff
@@ -21,7 +21,7 @@ howCharged:
     options:
       fixed: Fixed costs
       assessed: Assessed costs
-      both: Both
+      both: Both fixed and assessed costs
```

Comment:
This will alter the radio option that is available in the section. Is that intentional?

Reply:
OK It's fine as is